README.rst

```diff
@@ -42,6 +42,37 @@ To use it, just include *vault.server* in your *top.sls*, and configure it using
     service:
       type: systemd
 
+``vault.approle_config.sls``
+----------------------------
+
+With a fresh Vault server install, you would have to run approle_config state
+separately after manually initialising and unseal Vault and this cannot be
+avoided.
+
+::
+
+  vault:
+    version: 0.11.2
```

Comment:
Vault is version 1.0.0 maybe we should use that :-)

Reply:
cool! that was few days ago..